libraries/chain/include/eosio/chain/snapshot_detail.hpp

```diff
@@ -119,12 +119,14 @@ namespace eosio::chain::snapshot_detail {
       flat_map<block_timestamp_type, proposer_policy_ptr> proposer_policies;
       flat_multimap<block_num_type, finalizer_policy_tracker> finalizer_policies;
       uint32_t                                            finalizer_policy_generation;
+      digest_type                                         last_pending_finalizer_policy_digest;
 
       // from block_state
       std::optional<valid_t>                              valid;
 
       snapshot_block_state_v7() = default;
 
+      // When adding a member initialization here also update block_state(snapshot_block_state_v7) constructor
```

Comment:
Will some test or static_assert fail if `snapshot_block_state_v7` is forgotten to update?

Reply:
The `switchToNoBlockLogWithIrrModeSnapshot` test of `nodeos_irreversible_mode_test.py` failed.